libcxx/test/std/containers/sequences/deque/deque.modifiers/insert_iter_iter.pass.cpp

```diff
@@ -9,6 +9,9 @@
 // REQUIRES: long_tests
 // UNSUPPORTED: GCC-ALWAYS_INLINE-FIXME
 
+// This test chokes on the sanitizers during CI runs. It appears we can address most of this by simply enabling optimizations.
+// ADDITIONAL_COMPILE_FLAGS: -O1
```

Comment:
We should add this flag only when we're running the sanitizers. We can do that with

```
// ADDITIONAL_COMPILE_FLAGS(msan): -O1
// ADDITIONAL_COMPILE_FLAGS(tsan): -O1
```

Reply:
Honestly this test runs so much faster with optimizations, we should consider keeping it on. But I'm OK making the suggested changes too.